sirius-header.js

```diff
@@ -5,19 +5,18 @@ export class SiriusHeader extends HTMLElement {
             prefix = 'http://localhost:8080'
         }
 
-        this.innerHTML = `
-             <head>
-                <meta charset="utf-8" />
-                <title>
-                        Sirius - Supervision
-                </title>
-                <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover" />
-                <meta name="api-base-uri" />
-                <meta name="theme-color" content="blue" />
-                <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-            </head>
-        
-            <body class="govuk-template__body sirius-header">
+        let workflowUrl = '';
+        let supervisionUrl = '';
+
+        if (window.location.href.indexOf("/supervision/workflow") > -1) {
+            workflowUrl = `aria-current="page"`;
```

Comment:
Might be better to rename these vars to `isWorkFlowSelected` as the fact it had Url in it confused me, since the check is on the url but the contents of the var isn't.

You could also extract the logic into a function and then just call that from the HTML template, though this works fine for the limited current number of checks.

Reply:
I'll give it a rename but might just stick with the short version for now and refactor when we're more widely using this header if that's ok